cmake/modules/SwiftSupport.cmake

```diff
@@ -38,6 +38,9 @@ function(add_swift_target target)
     foreach(flag ${AST_CFLAGS})
       list(APPEND compile_flags -Xcc;${flag})
     endforeach()
+    if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
+      list(APPEND compile_flags -sdk;`xcrun --show-sdk-path -sdk macosx`)
+    endif()
```

Comment:
Can we use `CMAKE_OSX_SYSROOT` instead please?  Yes, its a terrible name, because it can be iOS, tvOS, or watchOS ... but, the point is that we can pass that along once and not re-evaluate it each time.  That saves compile time and results in a simpler command line.

Reply:
Why not CMAKE_DARWIN_SYSROOT as Darwin is a catch-all for macOS, iOS, tvOS and watchOS … just saying.